Sources/SnapshotTestingExtensions/SnapshotTestBase.swift

```diff
@@ -51,7 +52,7 @@ open class SnapshotTestBase: XCTestCase {
 
                 assertSnapshot(
                     matching: vc,
-                    as: .image(on: config.device, precision: imageDiffPrecision),
```

Comment:
we should run our test-suite against this change. I'm not sure if `.wait(for: 0.0, ...)` is equivalent to the old implementation (it's a new runloop, might change behaviour of the tests). Probably we could overload the whole `assertSnapshotDevices` function to avoid changing the timing of existing tests.

Reply:
Overloading is a good idea, I just implemented that